WordPress/WordPressTest/PostCardCellTests.swift

```diff
@@ -263,9 +264,21 @@ class PostCardCellTests: XCTestCase {
         XCTAssertFalse(postCell.separatorLabel.isHidden)
     }
 
-    func testShowsWarningMessageForFailedPublishedPosts() {
+    func testShowsChangesWillBeUploadedWarningForFailedPublishedPostsWithRemote() {
         // Given
-        let post = PostBuilder().published().with(remoteStatus: .failed).confirmedAutoUpload().build()
+        let post = PostBuilder(context).published().withRemote().with(remoteStatus: .failed).confirmedAutoUpload().build()
+
+        // When
```

Comment:
I think we can get rid of the Given/When/Then comments, I find them more intrusive than helpful. If our team agrees that we're following this pattern the line break should be enough. What do you think?

Reply:
If the `Given` statement is just one line or the test method is short, it's fine to leave it out. But as the lines get longer, it's harder to see which line is the part being tested. I think it's important in this case:

https://github.com/wordpress-mobile/WordPress-iOS/blob/2c6380a68b583dafe3eba8bb7cccaa69790bbf5b/WordPress/WordPressTest/ViewRelated/Post/Utils/PostNoticeViewModelTests.swift#L97-L115

The pattern acts as a reminder not to test too many scenarios in one method. And also, just to organize the statements more clearly. 

> If our team agrees that we're following this pattern

That could be added to the team contract. 😉